.github/workflows/tests.yml

```diff
@@ -218,6 +218,10 @@ jobs:
                   FAIL=true
                 fi
 
+                # XXX
+                find "$COVERAGE_DIR" -type f -exec ls -lh {} +
+                df -h .
+
```

Comment:
Obviously this is just debug, but noting it's giving errors on the non-coverage tests since `$COVERAGE_DIR` isn't set.

Reply:
Removed now. I had wondered if it was blowing up because the files were too big, but it turned out that wasn't it. It was OOM-ing because of some bad `collectCoverageFrom` in a few packages trying to process `wordpress/` and so on.